### ISSUE TITLE
Investigate pep8 W605 in account SettingsView

### ISSUE THREAD

**Issue by devenney** [oneirism/reverie#67]
## Steps to Reproduce

1. Remove the `# noqa: W605` comment from [`account/urls.py L15`](https://github.com/oneirism/reverie/blob/master/account/urls.py#L15).
2. Execute `pipenv run prospector`.

## Expected Outcome

No error reported in [`account/urls.py L15`](https://github.com/oneirism/reverie/blob/master/account/urls.py#L15).

## Actual Outcome

```
Messages                                              
========                                              
                                                      
account/urls.py                                       
  Line: 15                                            
    pep8: W605 / invalid escape sequence '\-' (col 22)
```

**Comment by devenney** [comment]
I suspect this is a false-positive and is actually caused by L173. The route regex should use raw string syntax as specified in the [Django docs](https://docs.djangoproject.com/en/2.1/ref/urls/#re-path), as follows:

`r''password/reset/confirm/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>.+)/$'`

**Comment by devenney** [comment]
Resolved by #72. Thanks @eyedeeess.
